atlas

```diff
@@ -155,63 +150,59 @@ parse() {
 }
 usage() {
 cat<<'GETOPTIONSHERE'
-Atlas is a CLI tool that has essentially one command: `atlas pull`
+Atlas is a CLI tool that has essentially one command: ``atlas pull``
 
-Configuration file:
+Configuration file
+******************
 
-    Atlas defaults to using a configuration file named `atlas.yml` placed
-    in the root directory. Configuration file:
+Atlas defaults to using a configuration file named ``atlas.yml`` placed
+in the root directory. Configuration file::
 
     pull:
       repository: <organization-name>/<repository-name>
       branch: <branch-name>
       directory: <repo-directory-name>:<local-dir-name> ...
       filter: <pattern> ...
 
-    Atlas can also use a configuration file in a different path using the `--config` flag
-    after `atlas`: `atlas pull --config config.yml`.
+Atlas can also use a configuration file in a different path using the ``--config`` flag
+after ``atlas``: ``atlas pull --config config.yml``.
 
-    Atlas can also be used without a configuration file by using the flags below after
-    `atlas pull`.
+Atlas can also be used without a configuration file by using the flags below after
+``atlas pull``.
 
 Positional arguments DIRECTORY MAPPINGS ...
+*******************************************
 
-   One or more directory map pair separated by a colon (:) e.g. FROM_DIR:TO_DIR.
+One or more directory map pair separated by a colon (:) e.g. FROM_DIR:TO_DIR.
 
-   The first directory (FROM_DIR) represents a directory in the git repository.
-   The second directory (TO_DIR) represents a local directory to copy files to.
+The first directory (FROM_DIR) represents a directory in the git repository.
+The second directory (TO_DIR) represents a local directory to copy files to.
 
-   At least one directory pair is required:
+At least one directory pair is required::
 
-     $ atlas pull frontend-app-learning/messages:learning-app frontend-lib-test/messages:test-lib
+    $ atlas pull frontend-app-learning/messages:learning-app frontend-lib-test/messages:test-lib
 
-   This syntax is inspired by `docker --volume from_dir:to_dir` mounting syntax.
-
-Options:
+This syntax is inspired by the ``docker --volume from_dir:to_dir`` mounting syntax.
 
-    `-r` or `--repository`:
-        A repository GitHub repo slug defaults to 'openedx/openedx-translations'.
+Options
+*******
 
-    `-b` or `--branch`:
-        The git branch defaults to 'main'.
+* ``-r`` or ``--repository``: A repository GitHub repo slug defaults to 'openedx/openedx-translations'.
 
-    `-f` or `--filter`
-       A comma-separated (or space-separated) list of patterns match files and sub-directories.
-       This is mainly useful to filter specific languages to download.
+* ``-b`` or ``--branch``: The git branch defaults to 'main'.
 
```

Comment:
Do we need these newlines in-between the bullet points? It looks quite odd in the terminal.

Reply:
I'll check them.

Reply:
I've reverted it back to the previous format. Now this shouldn't be a problem.